_docs/principles/accessibility.md

```diff
@@ -0,0 +1,216 @@
+
+### Alerts
+
+Alert bars are a great example of how multiple indicators can be used. If you are indicating an error alert (for example, making it red) always include a status icon and alert description so that someone who is color blind can still see the problem.
+
+Other optional indicators to consider are:
+
+- **Alert type:** An optional alert title descriptor.
+- **Alert action link:** Provides a user with the next step of a process.
+- **Close action:** Closes the alert.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+{% figure [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/overview/accessibility/accessibility-alerts.svg)
+{% endfigure %}
+
+{% endcolumn %}
+</div>
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Color and contrast
+
+Readability requires sufficient contrast between the text and its background. Low contrast is especially problematic for users with low vision who also often have contrast sensitivity. To ensure text is readable, Helix adheres to accessibility standards defined by [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/TR/WCAG21/). These standards are based on contrast ratios calculated from text size, text color, and background color.  
+
+Helix minimum contrast standards require: 
+
+- Small text should have a contrast ratio of at least 4.5:1 against its background color.
+- Large text (at 14px bold/18px regular and up) should have a contrast ratio of at least 3:1. 
+- We recommend using the contrast ratios for icons or other critical elements.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+{% figure [caption: "Helix typography styles with their corresponding contrast ratios. All ratios were calculated using #fafafa (gray 25) background color."] [class:"image bg-light border"] %}
+![]({{site.urll}}/assets/images/overview/accessibility/accessibility-typography.svg)
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+<div class="hxRow">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+### Use headings and spacing to group related content
+
+The use of space in visual design provides for designs that are both clear and pleasing to the eyes. Space creates relationships, defines hierarchies, and emphasizes content. Always keep these in mind when considering how to reduce visual noise and create visual flow in your design. Some people with cognitive disabilities find it difficult to read text when the lines are close together. Providing extra space between lines and paragraphs allows them to better track the next line and to recognize when they have reached the end of a paragraph. WCAG prescribes that line spacing (leading) is at least space-and-a-half within paragraphs, and paragraph spacing is at least 1.5 times larger than the line spacing.
```

Comment:
For the last sentence in this line, avoid starting a sentence with an acronym. Consider `The WCAG prescribes` as one fix.

Reply:
updated with link to https://www.w3.org/TR/WCAG20/